superset/migrations/versions/3ebe0993c770_filterset_table.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add filter set model
+
+Revision ID: 3ebe0993c770
+Revises: 07071313dd52
+Create Date: 2021-03-29 11:15:48.831225
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "3ebe0993c770"
+down_revision = "07071313dd52"
+
+import sqlalchemy as sa
+from alembic import op
+
+
+def upgrade():
+    op.create_table(
+        "filter_sets",
+        sa.Column("created_on", sa.DateTime(), nullable=True),
+        sa.Column("changed_on", sa.DateTime(), nullable=True),
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("name", sa.VARCHAR(500), nullable=False),
+        sa.Column("description", sa.Text(), nullable=True),
+        sa.Column("json_metadata", sa.JSON(), nullable=False),
```

Comment:
We should avoid including the type as part of the column/variable name. Also, if possible can we find a more descriptive name than `metadata`? I'm not sure what all is contained in `metadata`. Perhaps `configuration` or `definition`?

Reply:
Ok so a bit context here .
The purpose of this field is for storing the list of native-filters config as they are stored today under the Dashboard metadata.

About the field name , I'm ok with anything we decide .

Let's see what others like .
@villebro any thoughts?

Reply:
I suggest we not use `sa.JSON()` here but rather go with `sa.Text()` as we have until now, as with `sa.JSON()` we're not sure how it behaves on various metadata engines. So let's just keep it `Text()` and assume we need to parse/format when deserializing/serializing.

Reply:
Dashboard model uses JSON as well and it is very convenient to use... If I expect to Json object why not use JSON?
why do you use int and not everything as string? 
Please explain to us why JSON is bad practice? how are you sure Text is ok? If you trust sqlalchemy you should use their interface.